pkg/server/service_unix.go

```diff
@@ -34,7 +37,7 @@ const networkAttachCount = 2
 func (c *criService) initPlatform() error {
 	var err error
 
-	if runcsystem.RunningInUserNS() {
```

Comment:
I think we can safely keep this

Reply:
Ah, I saw this is being moved into https://github.com/containerd/containerd/pull/4230 . That's fine to me as well.